build-system/tasks/json-check.js

```diff
@@ -21,6 +21,33 @@ const log = require('fancy-log');
 const through2 = require('through2');
 const {jsonGlobs} = require('../config');
 
+const expectedCaches = ['cloudflare', 'google'];
+
+/**
+ * Fail if caches.json is missing some expected caches.
+ */
+function checkCachesJson() {
+  return gulp.src(['caches.json',])
+      .pipe(through2.obj(function(file) {
+        try {
+          const obj = JSON.parse(file.contents.toString());
+        } catch (e) {
+          // Do nothing since this is handled in checkValidJson.
```

Comment:
It might make sense to at least log this as a warning. e.g., `yellow('Could not parse caches.json. This is most likely a fatal error that will be found by checkValidJson')`

Reply:
good idea, done